themes/default/content/docs/intro/concepts/stack.md

```diff
@@ -32,18 +32,26 @@ $ pulumi stack init broomllc/staging
 
 Fully qualified stack names also include the project name, in the form `orgName/projectName/stackName`, and this fully-qualified format is required in some contexts. In most contexts, the shorthands `orgName/stackName` or `stackName` are valid and use the default organization and the current project context.
 
-{{% notes "info" %}}
+{{% notes type="info" %}}
 While stacks with applied configuration settings will often be accompanied by `Pulumi.<stack-name>.yaml` files, these files are not created by `pulumi stack init`. They are created and managed with [`pulumi config`]({{< relref "/docs/reference/cli/pulumi_config" >}}). For information on how to populate your stack configuration files, see [Configuration]({{< relref "/docs/intro/concepts/config" >}}).
 {{% /notes %}}
 
-## Deploy a project
+## Plan a deployment
```

Comment:
Is this how we refer to this? I ask because at least to me, it'd be slightly clearer say "Generate an update plan" here.

```suggestion
## Generate an update plan
```

I'd also suggest moving these both down a bit further, maybe just below select, as the order we've got now is already somewhat chronological/linear: -- "create", "list", "select" -- so "update" would seem right at home after those. What do you think?